public/app/features/alerting/unified/rule-list/GrafanaRuleLoader.tsx

```diff
@@ -30,8 +36,33 @@ export function GrafanaRuleLoader({ rule, groupIdentifier, namespaceName }: Graf
     return <AlertRuleListItemLoader />;
```

Comment:
I know this is not related to this PR, but I found this component name confusing, given we use xxxLoader naming in this context to the actual load of something. And in this case, this `AlertRuleListItemLoader` is not loading anything but showing ListItem as an skeleton. Maybe we could rename it as `SkeletonRuleListItem`  or something like that?